Add guidance for break-word override class (#3968)
alphagov/govuk-design-system: src/styles/font-override-classes/index.md

You can also use tabular numbers for numbers that dynamically update, such as in a counter. The equal width of each digit prevents the numbers from visually moving towards and away from each other as the counter updates.

Activate tabular numbers by using `govuk-!-font-tabular-numbers`.

> ## Breaking up long words
> 
> You may be required to display long words that cannot be changed or broken into smaller parts, such as:
> 
> - technical or scientific terms
> - long email addresses
> - words from other languages
> 
> Long words can create issues in constrained spaces such as mobile device screens and data tables. They can 'break out' of the layout, resulting in a broken visual appearance and requiring users to scroll horizontally to view all of your content.
> 
> You can help to mitigate these issues by surrounding content likely to 'break out' with `govuk-!-text-break-word`.
> 
> This class forcibly splits long words onto multiple lines when they become longer than the width of the container. It does so indiscriminately, which may make compound words more difficult to read. You can control where these breaks will occur by inserting [the `wbr` HTML tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/wbr) into your content.

Reviewer: Thanks beeps! I've suggested a few more changes for style.
Still trying to find a better word for 'indiscriminately'.

```suggestion
## Breaking up long words

You might need to show long words that cannot be changed or broken into smaller parts, such as:

- technical or scientific terms
- long email addresses
- words from other languages

Long words can create issues in constrained spaces such as mobile device screens and data tables. They can 'break out' of the layout, resulting in a broken visual appearance and requiring users to scroll horizontally to view all of your content.

You can help to reduce these issues by surrounding content likely to 'break out' with `govuk-!-text-break-word`.

This class forcibly splits long words onto multiple lines when they become longer than the width of the container. It does so indiscriminately, which might make compound words more difficult to read. You can control where these breaks will occur by inserting [the `wbr` HTML tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/wbr) into your content.
```
